src/processor/operator/persistent/writer/parquet/column_writer.cpp

```diff
@@ -50,6 +50,7 @@ std::unique_ptr<ColumnWriter> ColumnWriter::createWriterRecursive(
     }
     auto schemaIdx = schemas.size();
     switch (type->getLogicalTypeID()) {
+    case LogicalTypeID::UNION:
```

Comment:
I think it's better to separate this change to another PR for reading and writing UNION. so we can properly test it.